kie-server-parent/kie-server-tests/pom.xml

```diff
@@ -944,6 +913,14 @@
           <groupId>io.netty</groupId>
           <artifactId>netty-transport-native-epoll</artifactId>
         </dependency>
+          <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-codec-socks</artifactId>
+          </dependency>
+          <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-handler-proxy</artifactId>
+          </dependency>
```

Comment:
Hello @lucamolteni See these deps we had to add for Kie Server JMS tests - these deps may also be required with KS JMS client (or netty-all)

Reply:
Ok Thanks @jesuino. Are the JMS tests failing? Or are they running without these